Mono.CairoWarp/Properties/AssemblyInfo.cs

```diff
@@ -32,6 +32,6 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.0.0.0")]
-[assembly: AssemblyFileVersion("0.0.0.0")]
-[assembly: AssemblyInformationalVersion("VERSION_STRING")]
+// [assembly: AssemblyVersion("0.0.0.0")]
+// [assembly: AssemblyFileVersion("0.0.0.0")]
+// [assembly: AssemblyInformationalVersion("VERSION_STRING")]
```

Comment:
We want to keep this symbols, if those are provided now from csproj, this file can probably be removed.

Reply:
Yeah, can be removed because this stuff is now in csproj file
let me do